### DISCUSSION TITLE
Multiple Links

### DISCUSSION THREAD

**Discussion by Hawkins1212** [Tzahi12345/YoutubeDL-Material#466] [Ideas] [answered]
It would be useful in the app if it were possible to add multiple links at one time.  For example, I go to a streaming service and use LinkGopher extension to pull the links of all the videos of a certain season at the same time.  The output is copied and pasted like this:

https://www.streamingservice.com/video/Video1
https://www.streamingservice.com/video/Video2
https://www.streamingservice.com/video/Video3
https://www.streamingservice.com/video/Video4
https://www.streamingservice.com/video/Video5
https://www.streamingservice.com/video/Video6
https://www.streamingservice.com/video/Video7

It'd be nice to be able to just paste that into straight to the input and the program would recognize seven different videos and capture them all.  Something similar to a text box input where all new line items represent a new video link.  

Thanks in advance!

**Comment by Tzahi12345** [comment, marked as answer]
So there has been an issue (#119) for this, but since this is the second request (third if you include #190), I decided to go ahead and implement it.

Here's a screenshot:

![image](https://user-images.githubusercontent.com/707413/135199318-65456cb5-b221-4069-bdf6-f98e94c6243c.png)

Thanks for the suggestion! You can use this in the latest `nightly`.

**Reply by Hawkins1212** [reply]
Thanks so much!!! This is Great!!

**Reply by Hawkins1212** [reply]
In attempting to utilize, I am copying and pasting from seperate lines in Notepad; however, the text is just showing up in linear format (not separate lines as your screenshot indicates).  I updated and am running newest version:
Installed version: v4.2 - done  You are up to date.
Installation type: docker
Docker tag: nightly
Commit hash: 69767a8
Build date: 2021-09-29

This is the "input" I am copying:
![image](https://user-images.githubusercontent.com/87832357/135204534-b4280988-eea4-475a-8032-7a4d5d3061c9.png)

And this is how it reflects in the program:
![image](https://user-images.githubusercontent.com/87832357/135204610-1626d0eb-2a30-4d99-b84a-7ed509b0d780.png)

Am I missing something?  I even tried shift+enter

**Reply by Tzahi12345** [reply]
Looks like you were a bit too early on the pull, I should've mentioned it takes an hour and a half or so for the nightly to update. Try pulling again and it should work!

**Reply by Hawkins1212** [reply]
You are right.  My Unraid showed an update so I figured that was it.  Just checked again -- another update and now it works like a charm.

Thanks so much.  This is AWESOME!
